AK/JsonParser.cpp

```diff
@@ -11,6 +11,10 @@
 
 namespace AK {
 
+static constexpr bool is_space(int ch) {
```

Comment:
Please pay attention to the different number types (especially signed vs unsigned). Also, I think (but am not sure), the `static` can be omitted here because the method is `constexpr`.
```suggestion
constexpr bool is_space(char ch) {
```